beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -99,14 +99,14 @@ func (vs *Server) GetBeaconBlock(ctx context.Context, req *ethpb.BlockRequest) (
 	}
 
 	resp, err := vs.BuildBlockParallel(ctx, sBlk, head, req.SkipMevBoost, builderBoostFactor)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not build block in parallel")
+	}
 	log.WithFields(logrus.Fields{
 		"slot":               req.Slot,
 		"sinceSlotStartTime": time.Since(t),
 		"validator":          sBlk.Block().ProposerIndex(),
 	}).Info("Finished building block")
```

Comment:
This is a valuable log about the timing. Why not add a field with the error? Then you'll know if it was successful or not while maintaining the log about the timing.

```suggestion
	if err != nil {
		return nil, errors.Wrap(err, "could not build block in parallel")
	}
	log.WithFields(logrus.Fields{
		"slot":               req.Slot,
		"sinceSlotStartTime": time.Since(t),
		"validator":          sBlk.Block().ProposerIndex(),
		"error": err,
	}).Info("Finished building block")
```

Suggestion needs formatting via gofmt

Reply:
@prestonvanloon It seems like printing nil-able error will mess the logs, so I decided to log for failure by wrapping `err`. Could you please review this one?

Reply:
Why would it mess up the logs? If error is nil then you will see `error=nil` in the logs

With your fix, we still lose the timing since I would have to search for an entirely different log message when there is a failure rather than filtering for a non-nil error

Reply:
@prestonvanloon
Agreed. Adding an field with error can be sufficient. Thanks!